static/style.css

```diff
@@ -1,5 +1,381 @@
 
+:root {
```

Comment:
This CSS file is very long. Split it into more CSS files so that it's easier to read.